src/store/RootState.ts

```diff
@@ -1,4 +1,5 @@
 export default interface RootState {
     user: any;
     product: any;
+    orders: any;
```

Comment:
```suggestion
    order: any;
```

Reply:
Sir, Changed the module name in rootstate.